rules/example.ts

```diff
@@ -9,7 +9,7 @@ const STATE = z.object({
 export default defineRule({
   id: 'example',
   load: () => STATE.parse({ foo: 4 }),
-  check: async ({ api }) => {
+  check: ({ api }) => {
```

Comment:
I now allow this to be `sync`.

This is because `Deno`'s `lint` doesn't like the keyword `async` being used without `await`.
So sure. it was a relatively easy change to make.